public/app/features/dashboard/state/analyticsProcessor.ts

```diff
@@ -18,7 +18,12 @@ export function emitDataRequestEvent(datasource: DataSourceApi) {
       return;
     }
 
-    if (data.request.exploreMode) {
```

Comment:
Just for curiosity: Is this field no longer user? 🤔  Thanks!

Reply:
This field is used in Explore but it seems that it's not set all the time, so it's not reliable enough to use it here.